docs/developer/testing/TESTING.md

```diff
@@ -6,56 +6,68 @@
 
 <!-- Can we add this as pre run for make test-e2e? -->
 ```bash
-go install -v github.com/onsi/ginkgo/v2/ginkgo
+go install -v -mod=mod github.com/onsi/ginkgo/v2/ginkgo
 ```
 
-### Setup backup storage configuration
+### AWS setup
 
-To get started, the test suite expects 2 files to use as configuration for
-Velero's backup storage. One file that contains your credentials, and another
-that contains additional configuration options (for now, the name of the
-bucket).
+> **Note:** If you are using IBM Cloud, follow this section.
 
-The default test suite expects these files in `/var/run/oadp-credentials`, but
-can be overridden with the environment variables `OADP_CRED_FILE` and
-`OADP_BUCKET_FILE`.
+To get started, you need to provide the following **required** environment variables.
 
-To get started, create these 2 files:
-`OADP_CRED_FILE`:
+| Variable | Description | Default Value | required |
+|----------|----------|---------------|---------------|
+| `OADP_CRED_FILE` | The path to credentials file for backupLocations | `/var/run/oadp-credentials/new-aws-credentials` | true |
+| `OADP_BUCKET` | The bucket name to store backups | `OADP_BUCKET_FILE` file content | true |
+| `CI_CRED_FILE` | The path to credentials file for snapshotLocations | `/Users/drajds/.aws/.awscred` | true |
+| `VSL_REGION` | The region of snapshotLocations | - | true |
+| `BSL_REGION` | The region of backupLocations | `us-east-1` | false |
+| `OADP_TEST_NAMESPACE` | The namespace were OADP will be installed (it must exist prior to run) | `openshift-adp` | false |
+
+The expected format for `OADP_CRED_FILE` and `CI_CRED_FILE` files is:
 ```
 [<INSERT_PROFILE_NAME>]
 aws_access_key_id=<access_key>
 aws_secret_access_key=<secret_key>
 ```
-If you use one profile name different from `default`, also change `BSL_AWS_PROFILE`.
-
-<!-- change to env var? -->
-`OADP_BUCKET_FILE`:
-```
-<bucket_name>
+> **Note:** If you use one profile name different from `default` for backupLocations, also change `BSL_AWS_PROFILE` environment variable. snapshotLocations must use `default` profile name.
+
+To set the environment variables, run
+```sh
+export OADP_CRED_FILE=<path_to_backupLocations_credentials_file>
+export OADP_BUCKET=<bucket_name>
+export CI_CRED_FILE=<path_to_snapshotLocations_credentials_file>
+export VSL_REGION=<snapshotLocations_region>
+# non required
+export BSL_REGION=<backupLocations_region>
+export OADP_TEST_NAMESPACE=<test_namespace>
```

Comment:
you can also just do `VAR1=val make test-e2e` which I feel is more verbose and less hidden, less likely to make a mistake assuming values are correct.

Reply:
Going to also add in the docs!